File: connectors/src/main/java/com/binance4j/margin/dto/PriceIndex.java
package com.binance4j.margin.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * A symbol price index.
 * 
 * @property calcTime Last price calculation timestamp in ms.
 * @property price    Price.
 * @property symbol   Asset pair.
 */
@ApiModel("A symbol price index.")
data class PriceIndex(
@ApiModelProperty("Last price calculation timestamp in ms.")
@JsonProperty("calcTime") var calcTime: Long = 0L,
@ApiModelProperty("Price.")
@JsonProperty("price") var price: String = "",
@ApiModelProperty("Asset pair.")
@JsonProperty("symbol") var symbol: String = "")
{
}

File: connectors/src/main/kotlin/com/binance4j/margin/dto/PriceIndex.kt
package com.binance4j.margin.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * A symbol price index.
 * 
 * @property calcTime Last price calculation timestamp in ms.
 * @property price    Price.
 * @property symbol   Asset pair.
 */
@ApiModel("A symbol price index.")
data class PriceIndex(
@ApiModelProperty("Last price calculation timestamp in ms.")
 val calcTime : Long,
@ApiModelProperty("Price.")
 val price : String,
@ApiModelProperty("Asset pair.")
 val symbol : String)
{
}
